README.md

```diff
@@ -12,39 +12,27 @@ It also optionally supports subscription/unsubscription through a SaaS product o
 
-### List your products (APIs/Usage Plans)
-
-Add your API Gateway Usage Plans and APIs to `lambdas/backend/catalog/index.js`, using the format below, and add your Swagger files to `lambdas/backend/catalog/` directory. If you have not yet created an API and Usage Plan, see [Generate Your Own API Gateway Developer Portal](https://aws.amazon.com/blogs/compute/generate-your-own-api-gateway-developer-portal/) for a detailed walkthrough. Alternatively, skip this step for now if you just want to get started with your developer portal (A placeholder API with swagger definition is provided for you for demonstration purposes, however, some features such as __Subscribe__ will not work)
-
-```json
-{
-  "id": "YOUR_USAGE_PLAN_ID",
-  "name": "Free",
-  "apis": [{
-    "id": "YOUR_API_ID",
-    "image": "/sam-logo.png",
-    "swagger": petStoreSwaggerDefinition
-  }]
-}
-```
-
-__TIP:__ If you put your api product images in the `dev-portal/public` directory, you can simply do `"image": "/your-api-product-image.svg"`. `image` is also optional.
-
-Add your swagger definition to the `swagger` property to enable documentation for your API.
+Then, ensure that you have an S3 bucket to put zipped lambda functions into. It can be private, and will be referred to in this readme as "your-lambda-artifacts-bucket-name".
 
 ### Setup and deploy
 
 Run:
 
 ```bash
-npm run setup# Window users: all commands must use the `win-` prefix version, eg. npm run win-setup
+sam package --template-file ./cloudformation/template.yaml --output-template-file packaged.yaml --s3-bucket your-lambda-artifacts-bucket-name
 ```
 
-Follow the prompts and enter your region, and names for your S3 buckets and CloudFormation stack. The names you provide for the S3 buckets must be unique to that region (ie. not just unique to your account) so it is recommended to add a prefix or suffix (eg. my-org-dev-portal). You can choose to provide an existing bucket for the __artifacts__ S3 bucket name, or a new one (in which case it will be created for you). The __site__ S3 bucket must __NOT__ exist, as this is managed by the CloudFormation stack.
+Followed by:
+
+```bash
+sam deploy --template-file ./packaged.yaml --stack-name "dev-portal" --capabilities CAPABILITY_NAMED_IAM --parameter-overrides DevPortalSiteS3BucketName="dev-portal-static-assets" ArtifactsS3BucketName="dev-portal-artifacts"
+```
 
-After this command completes, it will open your developer portal web app in your default browser.
+The command will exit when the stack creation is successful. If you'd like to watch it create in real-time, you can log into the cloudformation console.
+
+You can override any of the parameters in the template using the `--parameter-overrides key="value"` format. This will be necessary if you intend to deploy several instances of the developer portal.
```

Comment:
Documentation of common parameters to override?

Reply:
Will likely tell users to just read `template.yaml`. We could include a list/table of parameters, too, if we wanted.